src/System.Net.Http.WinHttpHandler/src/System/Net/Http/WinHttpHandler.cs

```diff
@@ -1701,7 +1146,7 @@ private static void SetNoClientCertificate(SafeWinHttpHandle requestHandle)
                 0);
```

Comment:
Nit: (line 1145) WINHTTP_NO_CLIENT_CERT_CONTEXT (equivalent to IntPtr.Zero) for completeness.